README.md

```diff
@@ -1 +1,47 @@
-# Library
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Javascript Library
+
+> The idea for the project is to make a small library with information collected from the user.
+![screenshot](https://cdn0.iconfinder.com/data/icons/education-and-learning-10/64/Books-512.png)
+
+The user can mark a book as read or they can delete it from the list completely. Each book is shown in its own card with the buttons for mark as read and for delete the book.
+
+## Built With
+
+- Javascript,
+- HTML-CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/pastorp3/Library/feature/index.html)
```

Comment:
- **_Improve the readme instructions: include how to setup the project locally_**